CRM/Civicase/APIHelpers/CaseDetails.php

```diff
@@ -227,7 +227,6 @@ private static function getRoleActivityFilter($contactId) {
         ON civicrm_activity.id = civicrm_activity_contact.activity_id
         WHERE civicrm_activity.is_deleted = 0
           AND civicrm_activity.is_current_revision = 1
-          AND civicrm_activity.is_test = 0
```

Comment:
why note remove similar queries from the FE as well?

* https://github.com/compucorp/uk.co.compucorp.civicase/blob/master/ang/civicase/activity/feed/directives/activity-feed.directive.js#L423
* https://github.com/compucorp/uk.co.compucorp.civicase/blob/master/ang/civicase/case/factories/get-case-query-params.factory.js#L51-L71
* https://github.com/compucorp/uk.co.compucorp.civicase/blob/master/ang/civicase/dashboard/directives/dashboard-tab.directive.js#L31-L71